Automation/create_vnet_script.ps1

```diff
@@ -0,0 +1,26 @@
+# to run this, navigate to the repo and run ./Automation/create_vnet_script.ps1 
```

Comment:
I think it may make sense to put these files into a separate Vnet folder within the Automation folder.

Reply:
Yeah I saw you do that for the sg, I thought it looked like a good idea - we should all agree on the same structure though